New version: SimpleSDMLayers v0.9.6

JuliaRegistrator opened the pull request:
- Registering package: SimpleSDMLayers
- Repository: https://github.com/PoisotLab/SpeciesDistributionToolkit.jl
- Created by: @tpoisot
- Version: v0.9.6
- Commit: 0c5cf92321527e4a1405f29394b7f54a68ac9392
- Reviewed by: @tpoisot
- Reference: https://github.com/PoisotLab/SpeciesDistributionToolkit.jl/commit/0c5cf92321527e4a1405f29394b7f54a68ac9392#commitcomment-132147607
- Description: Work with species distributions in Julia
<!-- 4bf6357f6f90d079a2c82a82d4628be430431ee706c79f2e8c88cfb5fd8b856bc0e9f0d92300ceb992a30e0eac7ab624161df9c5edb4d9191bfe70d8efa8919ab4b5071f48a22a736787b4320ea464d379b3ebf5979274f491c1c226018b6766617372afb2d311851a4f3fbb65ba7634db40e0e015abc4a16119a9fc114c6204ae8908a3b7e944745d83cedd8b7f39fa42a46fd0eca2666bd6fa23d51347d36d447f1b4cd2ef4fc02f774b5afa4f4e368f8270d75e221fab8dd289d57931bc05c6e829bfb568917766a506a2c456ffdcc91906d9e60906a0634c22a4263e0196c8a48df0a156bdc70ebf355987c31000653d1bb540bc10070a3fd2501f41399b -->

tpoisot commented:
This should be closed in favor of #93420 (I thought the registrator would update this PR, sorry)

JuliaRegistrator committed:
New version: SimpleSDMLayers v0.9.6

UUID: 2c645270-77db-11e9-22c3-0f302a89c64c
Repo: https://github.com/PoisotLab/SpeciesDistributionToolkit.jl.git
Tree: 2852048eb658950e5e000a68fdad3a118e76fbae

Registrator tree SHA: 1ede4be248caa0b367cc8ebce8a0c76cb916c479
